src/js/lib.js

```diff
@@ -381,21 +367,21 @@ vjs.USER_AGENT = navigator.userAgent;
  * @constant
  * @private
  */
-vjs.IS_IPHONE = (/iPhone/i).test(vjs.USER_AGENT);
-vjs.IS_IPAD = (/iPad/i).test(vjs.USER_AGENT);
-vjs.IS_IPOD = (/iPod/i).test(vjs.USER_AGENT);
-vjs.IS_IOS = vjs.IS_IPHONE || vjs.IS_IPAD || vjs.IS_IPOD;
+var IS_IPHONE = (/iPhone/i).test(USER_AGENT);
+var IS_IPAD = (/iPad/i).test(USER_AGENT);
+var IS_IPOD = (/iPod/i).test(USER_AGENT);
+var IS_IOS = IS_IPHONE || IS_IPAD || IS_IPOD;
```

Comment:
I like the definition closer to the export, though `export let` looks crazy to my es5 eyes.